awd/cli.py

```diff
@@ -158,6 +158,7 @@ def deposit(
             return  # Unable to read DynamoDB table, exit
         for doi in dois:
```

Comment:
I would propose a line break above `for doi in dois:`.  This kicks off the main loop of the function, but it's kind of buried here.  I find myself coming back to this to orient.